### PR TITLE
Add session expired state to vpn's purchased state

### PR DESCRIPTION
**Pull request by simonhong**
Resolves https://github.com/brave/brave-browser/issues/28811

image

* Toggle button is disabled in session expired state.
* Expired state is cleared after 30days passed.
* Brave Account link in above msg will load https://account.brave.com/account in new tab

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

`BraveVPNServiceTest.LoadPurchasedStateSessionExpiredTest`

1. Enable vpn feature and make purchased state with @brave.com account with promotion
2. Check panel has main view(toggle button)
3. Cancel plan and check `canceled` state from `skus.state` in brave://local-state/
4. Modify os system time more than expired date in `skus.state`
   ex) `skus.state` has `"expires_at\":\"2023-04-09T09:51:57.938456\"` so set system's date to `2023-04-10`
5. Click vpn button and checks above expires view is shown
6. Modify os system's date to `2023-05-11` to invalidate  expired state
7. Click vpn button and check vpn panel shows sell view 

NOTE: When adjust os time, sometimes skus service gives invalid credential summary or 400 error when asking it. Because of this, vpn panel could show `INVALID` state instead of this expired msg. I think this due to manual time manipulation. I assume it'll work well with real world scenario.

### CODE REVIEW

**Review by spylogsster — APPROVED** [review]

File: components/resources/brave_vpn_strings.grdp [hunk 1]
```diff
@@ -258,4 +258,10 @@ However, we noticed this helper service is not running - and we don't want to le
   <message name="IDS_BRAVE_VPN_SUPPORT_TIMEZONE" desc="Label on dialog to report bug to technical support ">
     Timezone:
   </message>
+  <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_TITLE" desc="Title text for session expired panel">
+    Session expired
+  </message>
+  <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_CONTENT" desc="Content text for session expired panel">
+    Please login to your <ph name="BRAVE_ACCOUNT_BEGIN">$1</ph>Brave Account<ph name="BRAVE_ACCOUNT_END">$2</ph> to refresh your VPN session.
```
Comment (stephendonner) [inline components/resources/brave_vpn_strings.grdp:265]: Nit: should be s/login/log in

/cc @rmcfadden3

Comment (simonhong) [reply components/resources/brave_vpn_strings.grdp:265]: Pushed https://github.com/brave/brave-core/pull/17576 :)

Comment (rmcfadden3) [reply components/resources/brave_vpn_strings.grdp:265]: Yeah, confirmed: this should say `log in` as two words.

**Comment by stephendonner** [comment]
## Verification `IN-PROGRESS` using 

Brave | 1.51.47 Chromium: 112.0.5615.29 (Official Build) nightly (64-bit)
-- | --
Revision | 8a59304c62e066962363d2fd8edd8e7b459c42bd-refs/branch-heads/5615@{#497}
OS | Windows 10 Version 22H2 (Build 19045.2728)

### Steps:
1. installed `1.51.47`
2. launched Brave
3. purchased and configured VPN
4. connected to `BraveVPN`

**Pull request merged**